tests/averaging/rotation/test_rotation_averaging_base.py

```diff
@@ -22,6 +23,54 @@ def setUp(self):
 
         self.obj = DummyRotationAveraging()
 
+    def test_circle_two_edges(self):
+        """Tests for 4 poses in a circle, with a pose connected to its immediate neighborhood."""
+
+        if isinstance(self.obj, DummyRotationAveraging):
+            self.skipTest("Test case invalid for dummy class")
+
+        i2Ri1_dict, wRi_expected = sample_poses.convert_data_for_rotation_averaging(
+            sample_poses.CIRCLE_TWO_EDGES_GLOBAL_POSES, sample_poses.CIRCLE_TWO_EDGES_RELATIVE_POSES
+        )
+        wRi_computed = self.obj.run(len(wRi_expected), i2Ri1_dict)
+        self.assertTrue(geometry_comparisons.compare_rotations(wRi_computed, wRi_expected))
```

Comment:
Can we somehow use a helper that takes in the relative and global poses and does the assertion? Because I feel that code is repeated a lot.

Reply:
i think having utils that return bools rather than assert things is more in-line with calling a utility in a unit test, though, so i kind of like it as it is

Reply:
I added Akshay's recommendation of creating a helper to run the averaging and assert results. It does cut down on duplicate logic, so I am keeping it for your review. We can roll it back too.